src/component/helper/RoamController.ts

```diff
@@ -242,6 +252,44 @@ class RoamController extends Eventful<{
         // their event both, and the final behavior is determined
         // by event listener themselves.
 
+        if (gestureOnTouchPad) {
+          const WindowScrollSpeedFactor = .3;
```

Comment:
We should probably make these parameters configurable in options.

Reply:
Yeah, Thanks a lot @Ovilia, I would like to add scrolling both directions as a feature. So I think this not a bug. the name `natureMoveOnMouseWheel` is good for me.

And also, I already set `gestureOnTouchPad` as an options adjust with 
```
zoomOnMouseWheel: false,
moveOnMouseWheel: true,
```
now it looks like:
```
zoomOnMouseWheel: false,
moveOnMouseWheel: true,
gestureOnTouchPad: true,
```
Users can set this option use `setOption();`

Reply:
> We should probably make these parameters configurable in options.

Sorry, I have little confuse,  which kind of options? Is the options of `InsideZoomModel` ?